soh/soh/Enhancements/randomizer/3drando/location_access/locacc_castle_town.cpp

```diff
@@ -198,7 +198,7 @@ void AreaTable_Init_CastleTown() {
 
   areaTable[MARKET_SHOOTING_GALLERY] = Area("Market Shooting Gallery", "Market Shooting Gallery", NONE, NO_DAY_NIGHT_CYCLE, {}, {
                   //Locations
-                  LocationAccess(MARKET_SHOOTING_GALLERY_REWARD, {[]{return IsChild;}}),
+                  LocationAccess(MARKET_SHOOTING_GALLERY_REWARD, {[]{return IsChild && ChildWallet;}}),
```

Comment:
Just want to make sure you got all the locations that require money.  Scanning the location lists,  I don't see anything for Bombchu Bowling (which can just be added to the CanPlayBowling boolean), or for all of the Deku scrubs (which I don't know if they would fall under the shopping logic that uses SufficientWallet or not).

Reply:
CanPlayBowling has already been updated to require child wallet and I'm pretty sure the scrubs are covered by the shopping logic

Reply:
CanBuy, the part of logic that adds on a check for the price to shopping, only checks for Catagory::cShop and not Catagory::cDekuScrub, meaning they are NOT covered by shopping logic. As Scrubs had a price cap of 95 before, this wasn't a problem, but now it is.